dialogflow/stopwatch/Stopwatch/AppDelegate.swift

```diff
@@ -14,16 +14,32 @@
 // limitations under the License.
 //
 import UIKit
+import FirebaseFunctions
+import Firebase
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-  
   var window: UIWindow?
+  var hasRequestedForToken = false
```

Comment:
Is this variable used anywhere else?

Reply:
we are not using this variable anymore. Removing it.